README.md

```diff
@@ -1,61 +1,56 @@
 # webpack.js.org
 
-[![Build Status][13]][10]
-[![Standard Version][12]][11]
+[![Build Status][build-status]][build-status-url]
+[![Standard Version][release]][release-url]
 
 Guides, documentation, and all things webpack.
 
-
 ## Content Progress
 
 Now that we've covered much of the backlog of _missing documentation_, we are
 starting to heavily review each section of the site's content to sort and
 structure it appropriately. The following issues should provide a pretty good
 idea of where things are, and where they are going:
 
-- [Guides - Review and Simplify][1]
-- [Concepts - Review and Organize][2]
-- [API - v4 Rewrite][8]
+- [Guides - Review and Simplify][issue-1258]
```

Comment:
Why use numbers instead of `guides`, `concepts`, `api` like you did in the rest of the references?

Reply:
I was not sure between this & issue number.
Will change it :+1:

Reply:
@montogeek pls review the new changes.

Thank you